### PR TITLE
Dev/minify test

### PR DESCRIPTION
**Pull request by anonymoussprocket**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,34 +1,40 @@
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
-const path = require('path')
-const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
-const nodeExternals = require('webpack-node-externals');
```
Comment (anonymoussprocket) [inline webpack.config.js]: Why remove this?

Comment (developer0623) [reply webpack.config.js]: @anonymoussprocket  I used the webpack to compile the conseiljs.min.js.
We need to embed the min file using a script tag on an HTML file.
There is no node_module.
So we need to import node modules when compile.

File: tsconfig.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
     "declaration": true,                      /* Generates corresponding '.d.ts' file. */
     "sourceMap": true,                        /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-     "outDir": "dist",                        /* Redirect output structure to the directory. */
+     "outDir": "./dist/",                        /* Redirect output structure to the directory. */
```
[the remark this answers is not in the document]
Comment (developer0623) [reply tsconfig.json]: There is no need to change.

File: src/tezos/TezosOperations.ts [hunk 3]
```diff
@@ -52,7 +55,7 @@ export namespace TezosOperations {
         }
 
         const hexSignature: string = CryptoUtils.base58CheckEncode(opSignature, "edsig").toString();
-        const signedOpBytes = Buffer.concat([sodium.from_hex(forgedOperation), opSignature]);
+        const signedOpBytes = Buffer.concat([Buffer.from(sodium.from_hex(forgedOperation)), Buffer.from(opSignature)]);
```
[the remark this answers is not in the document]
Comment (developer0623) [reply src/tezos/TezosOperations.ts]: The result of `sodium.from_hex` is Unit8Array.
So there is an issue when using Buffer.concat.

File: webpack.config.js [hunk 4]
```diff
@@ -1,34 +1,40 @@
...
-    entry: "./src/index.ts",
-    target: 'node',
+    mode: 'production',
+    devtool: 'inline-source-map',
+    entry: './src/index-browser.ts',
     output: {
-        path: path.resolve(__dirname, './build'),
-        filename: 'bundle.js',
+        // filename: "bundle.js"
+        path: path.resolve(__dirname, './dist'),
```
Comment (anonymoussprocket) [inline webpack.config.js]: Why switch output directories?

File: src/utils/CryptoUtils.ts [hunk 5]
```diff
@@ -77,17 +77,17 @@ export function getBase58BytesForPrefix(prefix: string): Buffer {
     switch(prefix)
     {
         case "tz1":
-            return new Buffer([6, 161, 159]);
+            return Buffer.from([6, 161, 159]);
```
Comment (anonymoussprocket) [inline src/utils/CryptoUtils.ts]: Was `new Buffer()` causing problems?

Comment (developer0623) [reply src/utils/CryptoUtils.ts]: But `new Buffer` was deprecated.

File: tsconfig.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (anonymoussprocket) [reply tsconfig.json]: then revert please

File: package.json [hunk 6]
```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@ledgerhq/hw-transport": "4.21.0",
-    "@ledgerhq/hw-transport-node-hid": "^4.35.0",
+    "@ledgerhq/hw-transport-node-hid": "^4.24.0",
```
[the remark this answers is not in the document]
Comment (developer0623) [reply package.json]: I remoded

File: webpack.config.js (hunk 4 above)
Comment (developer0623) [reply webpack.config.js]: We used `build folder` before to test.
Currently. we use `dist folder` for npm and use webpack to minify the package.
It is better the min.js is in `dist`.
So I changed.

File: package.json [hunk 7]
```diff
@@ -2,14 +2,15 @@
...
+  "browser": "dist/index-browser.js",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "test": "mocha test/*.spec.ts",
     "integration-test": "mocha --compilers ts-node/register --require source-map-support/register --require ts-node/register --full-trace --timeout 10000 integration_test/*.spec.ts",
     "format": "eslint --fix-dry-run src/*/*.ts",
-    "package": "rm -rf ./dist && tsc",
-    "build": "webpack && rm -rf ./build/dist",
+    "package": "rm -rf ./dist && tsc && yarn build",
```
[the remark this answers is not in the document]
Comment (developer0623) [reply package.json]: It was changed to `npm`.

File: src/utils/ConseilQuery.ts [hunk 8]
```diff
@@ -56,7 +56,7 @@ function sanitizeFilter(filter: TezosFilter): TezosFilter {
         block_protocol: filter.block_protocol.filter(Boolean),
         operation_id: filter.operation_id.filter(Boolean),
         operation_source: filter.operation_source.filter(Boolean),
-        operation_destination: filter.operation_source.filter(Boolean),
+        operation_destination: filter.operation_destination.filter(Boolean),
```
Comment (anonymoussprocket) [inline src/utils/ConseilQuery.ts]: good catch

**Pull request closed without merging**